templates/conf/kubernetes/cluster-autoscaler.conf.erb

```diff
@@ -3,9 +3,9 @@
   @id in_tail_cluster_autoscaler
   multiline_flush_interval 5s
   path /var/log/cluster-autoscaler.log
-  pos_file /var/log/fluentd-cluster-autoscaler.log.pos
+  pos_file "#{'/var/log/' + ENV.fetch('FLUENT_POS_PREFIX', '') + 'fluentd-cluster-autoscaler.log.pos'}"
```

Comment:
How about using File.join to simplify to represent what you wanted to do like as?

```
pos_file "#{File.join('/var/log/', ENV.fetch('FLUENT_POS_PREFIX', ''), 'fluentd-cluster-autoscaler.log.pos')}"
```

Just joining as String instances is dangerous. Safely joining method should be needed to use here.

Reply:
I just tested a bit. Using File.join will create an addition directory. Not sure is that intention to have addition directory.
If that is the intention I can change to File.join.

---
And I just tested with `../..` with File.join. Look like it work same as string concat on this case. 

I set `FLUENT_POS_PREFIX` == `PREFIX` and `DFLUENT_POS_PREFIX` == `../../PREFIX`.

```
irb(main):001:0> File.join('/var/log/', ENV.fetch('FLUENT_POS_PREFIX', ''), 'fluentd-cluster-autoscaler.log.pos')
=> "/var/log/PREFIX/fluentd-cluster-autoscaler.log.pos"
irb(main):002:0> File.join('/var/log/', ENV.fetch('FLUENT_POS_PREFIX_NOT_EXIST', ''), 'fluentd-cluster-autoscaler.log.pos')
=> "/var/log/fluentd-cluster-autoscaler.log.pos"
irb(main):003:0> File.join('/var/log/', ENV.fetch('DFLUENT_POS_PREFIX', ''), 'fluentd-cluster-autoscaler.log.pos')
=> "/var/log/../../PREFIX/fluentd-cluster-autoscaler.log.pos"
```

Reply:
One more question. If added directory. Will fluentd create that directory?? Or will have exception??

Reply:
From the current Fluentd implementation, Fluentd will create directory if it doesn't exist:
https://github.com/fluent/fluentd/blob/master/lib/fluent/plugin/in_tail.rb#L239
This change will be safe.

Reply:
That's great. I modified to use File.join. And I changed the env name to `FLUENT_POS_EXTRA_DIR`. That should more fit for this behaviour.